graphdatascience/query_runner/protocol/write_protocols.py

```diff
@@ -94,3 +100,35 @@ def run_write_back(
             False,
             False,
         )
+
+
+class RemoteWriteBackV3(WriteProtocol):
+    def write_back_params(
+        self,
+        graph_name: str,
+        job_id: str,
+        config: Dict[str, Any],
+        arrow_config: Dict[str, Any],
+        database: Optional[str] = None,
+    ) -> CallParameters:
+        return RemoteWriteBackV2().write_back_params(graph_name, job_id, config, arrow_config, database)
+
+    def run_write_back(
+        self, query_runner: QueryRunner, parameters: CallParameters, yields: Optional[List[str]]
+    ) -> DataFrame:
+        def is_not_completed(result: DataFrame) -> bool:
+            result = write_fn().squeeze()
+            return result["status"] != Status.COMPLETED.name
+
+        @retry(retry=retry_if_result(is_not_completed), wait=wait_incrementing(start=0.2, increment=0.2, max=2))
```

Comment:
any particular reason for these retry values?

Reply:
Nope